packages/astro/test/fixtures/html-encoded-characters/src/pages/index.astro

```diff
@@ -0,0 +1,11 @@
+---
+---
+<html>
+<head><title>HTML Encoded Characters</title></head>
+<body>
+  <h1>&nbsp;&nbsp;&nbsp;Hello, world&semi;</h1>
+	<div>
+		<p>Nested elements&quest; No problem&period;&nbsp;</p>
```

Comment:
It looks like the parser handles these differently, so can you add a test for inside of an HTML attribute?

```
<p data-test="&quest;">
```